tests/integration/run.py

```diff
@@ -0,0 +1,984 @@
+        args.extend(self.options["command_args"])
+        return args
+
+
+class VerifyStepOptions(TypedDict):
+    """The configuration options of a verify step."""
+
+    main_args: Sequence[str]
+    command_args: Sequence[str]
+    database: str
+    policy: str | None
+    show_prelude: bool
+
+
+@dataclass
+class VerifyStep(Step[VerifyStepOptions]):
+    """A step running the ``macaron verify-policy`` command."""
+
+    @staticmethod
+    def options_schema(cwd: str) -> cfgv.Map:
+        """Generate the schema of a verify step."""
+        return cfgv.Map(
+            "verify options",
+            None,
+            *[
+                cfgv.Optional(
+                    key="main_args",
+                    check_fn=cfgv.check_array(cfgv.check_string),
+                    default=[],
+                ),
+                cfgv.Optional(
+                    key="command_args",
+                    check_fn=cfgv.check_array(cfgv.check_string),
+                    default=[],
+                ),
+                cfgv.Optional(
+                    key="database",
+                    check_fn=cfgv.check_string,
+                    default="./output/macaron.db",
+                ),
+                cfgv.Optional(
+                    key="policy",
+                    check_fn=check_required_file(cwd),
+                    default=None,
+                ),
+                cfgv.Optional(
+                    key="show_prelude",
+                    check_fn=cfgv.check_bool,
+                    default=False,
+                ),
+            ],
+        )
+
+    def cmd(self, macaron_cmd: str) -> list[str]:
+        """Generate the command of the step."""
+        args = [macaron_cmd]
+        args.extend(self.options["main_args"])
+        args.append("verify-policy")
+        args.extend(["--database", self.options["database"]])
+        args.extend(self.options["command_args"])
+        policy_file = self.options["policy"]
+        if policy_file is not None:
+            args.extend(["--file", policy_file])
+        show_prelude = self.options["show_prelude"]
+        if show_prelude:
+            args.extend(["--show-prelude"])
+        return args
+
+
+def gen_step_schema(cwd: str, check_expected_result_files: bool) -> cfgv.Map:
+    """Generate schema for a step."""
+    return cfgv.Map(
+        "steps[*]",
+        "name",
+        *[
+            cfgv.Required(
+                key="name",
+                check_fn=cfgv.check_string,
+            ),
+            cfgv.Required(
+                key="kind",
+                check_fn=cfgv.check_one_of(
+                    (
+                        "shell",
+                        "compare",
+                        "analyze",
+                        "verify",
+                    ),
+                ),
+            ),
+            cfgv.ConditionalRecurse(
+                condition_key="kind",
+                condition_value="command",
```

Comment:
I think this parameter should be `condition_value="shell"`.

Reply:
Good catch. Fixed in 46634c86ae3168ef7f997532bc0da0e5311bee44.